packages/use-midi/src/index.ts

```diff
@@ -39,7 +40,14 @@ interface Status {
 type Returns = [MidiMessage[], Status, RequestPermission, WebMidi.MIDIAccess?]
 
 const useMidi = (options: Options = {}): Returns => {
-  const { automaticallyRequestPermission = false, callback, debug = false, suppressActiveSensing = true, sysex = false } = options
+    const {
+    automaticallyRequestPermission = false,
+    callback,
+    debug = false,
+    suppressActiveSensing = true,
+    messageHistoryCount = 256,
```

Comment:
It's not like this is any popular and I'm not even sure if there is anyone besides you is using it, but this would be breaking (as in working different) change in my opinion. If we make it optional without a default value, we could bump only the minor version.

Reply:
So if I understand you correctly, you mean that I will not give by default a value for messageHistoryCount, so by default it work as it would normally does, and to limit the messages length, the user must explicitly set the option?

Reply:
Yeah that was my thinking, but I think you're right in providing a default, since the array filling up and crashing something isn't very desirable. So let's keep the default value and release it as `v2`.

Reply:
ok nice! **256** looks good to you for a default value?